### ISSUE TITLE
auto assign a targeting shape on initialization and show it large on the client screens

### ISSUE THREAD

**Issue by t3kt** [optexture/paint#12]
cat get credit for this idea.
"you are the _<color>_ square"
where color is the color that we assigned to you initially
see also optexture/vissom-client#1

**Comment by snoyface** [comment]
I have been also thinking we can just use numbers next to the circle/cross
hair. (That is the test case I used in the UX example I made) "you are
number N"

**Comment by JimAnkrom** [comment]
"You are the square" meaning we use geometric shapes? Or just the "You are
this color" notification - we did that in bleepout, was intending to do
that here too.

On Tue, Apr 19, 2016 at 10:07 AM, Jonathan Macleod <notifications@github.com

> wrote:
> 
> I have been also thinking we can just use numbers next to the circle/cross
> hair. (That is the test case I used in the UX example I made) "you are
> number N"
>

**Comment by t3kt** [comment]
Color is going to be variable, or at least customizable.
Having a shape will help differentiate people's markers from those of other people that happen to be using the same color. While there is still a chance that someone else will be using the same color and shape, it's a smaller chance than if color is the only factor.

**Comment by JimAnkrom** [comment]
Yah, this makes sense to me. Can we make shapes permanently assigned to
specific player channels? That way we can show the user their shape on
their screen.

**Comment by t3kt** [comment]
Yes. It would be assigned when they first connect and then fixed after that. We can convey that from sway to TD by sending a single osc message with the shape index when they first connect.

**Comment by JimAnkrom** [comment]
Ok, so to be clear, we want sway (or some client mechanism) to select the
shape, then push that to TD? Can do.

**Comment by t3kt** [comment]
yes. we can come up with a list of shapes and then send the index of the shape that sway assigned to the user.
